### DISCUSSION TITLE
Problem When Installing Directus

### DISCUSSION THREAD

**Discussion by rzkyyds** [domcloud/domcloud#62] [Q&A]
I tried to install Directus on the free plan with the template provided, but it always failed. The error line that appears is **ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @directus/app@10.9.1 build: `vite build`** Are there any problems?

![image](https://github.com/domcloud/domcloud/assets/23289851/539a38c9-09d4-49de-b699-962be6031f27)

**Comment by willnode** [comment]
This template [has been updated](https://github.com/domcloud/recipes/blob/master/boot-directus.yml). Please try to recreate the website.
